packages/wallet/src/services/TipTracker.ts

```diff
@@ -62,20 +66,34 @@ export class TipTracker extends PersistentDocumentTrackerSubject<Cardano.Tip> {
           connectionStatus$
         ]).pipe(
           // Throttle syncing by interval, cancel ongoing request on external trigger
+          tap(([isSettled, connectionStatus]) => {
```

Comment:
Some of these logs could also be added inline in the existing operators, but I think it's cleaner to keep them separate when possible